mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/ProxyConfigurationResourceDefinition.java

```diff
@@ -428,6 +429,36 @@ public PathAddress convertToTargetAddress(PathAddress aliasAddress, AliasContext
             return SimpleLoadProviderResourceDefinition.Attribute.FACTOR;
         }
 
+        AttributeValueTranslator SIMPLE_LOAD_FACTOR_DEFINED_VALIDATOR = new AttributeValueTranslator() {
```

Comment:
This property should be final.  Also, use camel case for non-static properties.
SIMPLE_LOAD_PROVIDER_TRANSLATION, however, should probably be both static and final.

Reply:
Ok, cannot be static here, so final will suffice.

Reply:
Yes, but it is allowed for SIMPLE_LOAD_PROVIDER_TRANSLATION.

Reply:
Got it now, yes it should, fixed.